torch/_prims_common/__init__.py

```diff
@@ -655,6 +655,12 @@ def compute_elementwise_output_strides(*tensors) -> tuple[int, ...]:
     if len(tensors) == 0:
         return ()
 
+    if len(tensors) == 1:
+        if torch._prims_common.is_non_overlapping_and_dense(tensors[0]):
```

Comment:
we prob should rename def is_non_overlapping_and_dense to 
is_non_overlapping_and_dense_or_false. its fine its reasonable definition of unbacked semantics its probably would be good to ensure inductor behave similarly with unbacked inputs.